Editor/HPUIInteractorPillEditor.cs

```diff
@@ -0,0 +1,65 @@
+using UnityEngine;
+using UnityEditor;
+using ubco.ovilab.HPUI.Interaction;
+
+namespace ubco.ovilab.HPUI.Editor
+{
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(HPUIInteractorPill), true)]
+    public class HPUIInteractorPillEditor : UnityEditor.Editor
+    {
+        private HPUIInteractorPill t;
+        private bool generating = false;
+        private float maxAngle = 90;
+        private float angleStep = 5;
+        private float axisA = 0.015f;
+        private float axisB = 0.015f;
+        private float axisC = 0.015f;
```

Comment:
Change th defaults to the values previously discussed

Reply:
done